roles/scalelite/templates/docker-compose.yml.j2

```diff
@@ -237,10 +265,10 @@ services:
       - RELATIVE_URL_ROOT=/greenlight
       - DEFAULT_LOCALE={{ scalelite_greenlight_default_locale }}
       - LOG_LEVEL={{ scalelite_greenlight_log_level }}
-      - REDIS_URL=redis://redis:6379
+      - REDIS_URL=redis://${REDIS_USER}:${REDIS_SECRET}@redis-greenlight:6379
```

Comment:
Scalelite and the Greenlight instances should use different users, or at least different passwords